.github/workflows/build_php71.yaml

```diff
@@ -0,0 +1,77 @@
+name: Rector Run
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - master
+
+jobs:
+    downgrade_php71:
+        runs-on: ubuntu-latest
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            # see https://github.com/shivammathur/setup-php
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.4
+                    coverage: none
+
+            # The Symfony Polyfill libraries are needed
+            -   run: composer require symfony/polyfill-php72 symfony/polyfill-php73 symfony/polyfill-php74 symfony/polyfill-php80 --no-install --no-update
+
+            # Execute the downgrade script
+            -   run: ./.github/workflows/scripts/downgrade_packages.sh 7.1
```

Comment:
@note Create GitHub Action with Rector parameter?

Reply:
Yeah! I'll work on that

Reply:
Do you want to create a new repo `rectorphp/rector-downgrade` to host the GitHub action?